### ISSUE TITLE
Change locales in Channels causes debug toolbar crash

### ISSUE THREAD

**Issue by jarekkowol** [Sylius/Sylius#8199]
First of all I added some locales in admin dashboard (pl_PL or pl). Next step is change in Channels.
I changed Default locale and Locale to Polish (Poland). This change causes debug toolbar crash. (404 not found in browser alert)

Back to the old settings fixes the problem.
Any idea?

**Comment by lchrusciel** [comment]
Did you ask for the help on our #support channel at slack? 

Also, did you also choose polish as in locales selectbox?
![zrzut ekranu 2017-06-26 o 09 07 44](https://user-images.githubusercontent.com/6213903/27528097-0eaa037a-5a4f-11e7-9b48-f489aae1d122.png)

**Comment by jarekkowol** [comment]
Thanks. If I select both it works correctly. When I deselect en_US I get an error

**Comment by lchrusciel** [comment]
I will keep it open to not forget about it before 1.0

**Comment by kochen** [comment]
@lchrusciel it's not only the debug toolbar, a lot is dependent on the (default) locale in the system and changing it to something other than `en_US` make the system unusable!
maybe this needs to have a higher priority...?

**Comment by lchrusciel** [comment]
@kochen we will take a look on this before 1.0

**Comment by kochen** [comment]
@lchrusciel any planned date for that release?
I believe this issue is bigger than "last moment".

**Comment by lchrusciel** [comment]
The roadmap and release date will be announced soon. I don't want to spoil :)

**Comment by h3llr4iser** [comment]
I have fixed the problem in my site, in my case I have deleted en_US from database and change locale in parameters.yml and I have the same error. By default, sylius instalation have 2 defaults parameters locales set to en_US. One is inside app/config/config.yml and the other in app/config/parameters.yml.
If you delete the one is in config.yml to get it from parameters you can set the locale to other different locale and the debug toolbar doesnt crash. the issue https://github.com/Sylius/Sylius/issues/8086 is similar

```
#app/config/config.yml

#parameters:
    #locale: en_US

```

```
#app/config/parameters.yml

parameters:
    locale: es_ES
```

**Comment by pjedrzejewski** [comment]
Fixed in #8602.

**Comment by pjedrzejewski** [comment]
Thanks for reporting!
